decimal.go

```diff
@@ -120,6 +120,9 @@ var (
 
 	// ErrZeroPowNegative is returned when raising zero to a negative power
 	ErrZeroPowNegative = fmt.Errorf("can't raise zero to a negative power")
+
+	// ErrExponentTooLarge is returned when the exponent is too large
+	ErrExponentTooLarge = fmt.Errorf("exponent is too large. Must be less than math.MaxInt32")
```

Comment:
```suggestion
	// ErrZeroPowNegative is returned when raising zero to a negative power
	ErrZeroPowNegative = errors.New("can't raise zero to a negative power")

	// ErrExponentTooLarge is returned when the exponent is too large
	ErrExponentTooLarge = errors.New("exponent is too large. Must be less than math.MaxInt32")
```

Please also fix the other ones that don't require formating

Reply:
Is there any performance penalty, or does it follow a standard? fmt.Errorf uses errors.New under the hood when the input is only a string. I assume they're almost identical.